docs/docs/filtering/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Overview
+sidebar_position: 1
+---
+
```

Comment:
I feel like some preamble here would be helpful, outlining what this new section is all about and what the value we can offer for the use case is.